infra/utils.py

```diff
@@ -78,11 +78,10 @@ def get_fuzz_targets(path):
   if not os.path.exists(path):
     return []
   fuzz_target_paths = []
-  for root, _, _ in os.walk(path):
-    for filename in os.listdir(path):
```

Comment:
Abhishek is right. 
The incorrect usage is in the call to `os.listdir`.
You should do `root, _, fuzzers in os.walk` then loop over the files and join with `root`.

Reply:
Noted. I was not aware there were cases where the fuzz targets were deeper than the current directory.